pkg/api/admin/platformworkloadidentityroleset_validatestatic.go

```diff
@@ -39,7 +39,7 @@ func (sv platformWorkloadIdentityRoleSetStaticValidator) validate(new *PlatformW
 		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "properties.openShiftVersion", "Must be provided")
 	}
 
-	if new.Properties.PlatformWorkloadIdentityRoles == nil || len(new.Properties.PlatformWorkloadIdentityRoles) == 0 {
+	if len(new.Properties.PlatformWorkloadIdentityRoles) == 0 || new.Properties.PlatformWorkloadIdentityRoles == nil {
```

Comment:
Perhaps omit one of these two checks - as far as I can tell there's no possible way with `[]struct{}` to distinguish between the zero-value being `nil` and the zero-value being an empty slice.

Reply:
addressed